code/drasil-printers/lib/Language/Drasil/Markdown/Helpers.hs

```diff
@@ -0,0 +1,112 @@
+-- | Defines helper functions for creating Markdown files.
+module Language.Drasil.Markdown.Helpers where
+
+import Prelude hiding ((<>), lookup)
+import Text.PrettyPrint (Doc, text, empty, (<>), (<+>), ($$), hcat, vcat)
+import Data.List.Split (splitOn)
+import Data.Map (lookup)
+import Language.Drasil.Printing.Helpers (ast)
+import Language.Drasil.Printing.LayoutObj (RefMap)
+
+data Variation =  Id | Align
+
+-- | Curly braces.
+br :: Doc -> Doc
+br x = text "{" <> x <> text "}"
```

Comment:
Are these functions duplicates of any in `Language.Drasil.Printing.Helpers`?

Reply:
The functions in `Language.Drasil.Printing.Helpers` are `String -> Doc`

https://github.com/JacquesCarette/Drasil/blob/6a86cda80c866f216d7cd663d4876850f4e971fe/code/drasil-printers/lib/Language/Drasil/Printing/Helpers.hs#L48-L53

I can either:
1) Keep it the way it is.
2) Remove the functions in `Language.Drasil.Markdown.Helpers` and use `show` on the parameter being passed in.

Which do you suggest?